src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/arm/reduce.cpp

```diff
@@ -18,6 +18,8 @@ namespace {
 
 std::vector<std::vector<ov::test::InputShape>> inputShapes_5D = {
     {{{}, {{2, 19, 2, 2, 9}}}},
+    {{{}, {{0, 19, 2, 2, 9}}}},
+    {{{}, {{1, 0, 0, 2, 9}}}},
```

Comment:
The tests are failing with
`Supported Reduce executor is not found` or ` primType is unexpected : undef_f32 Expected : acl_f32`
It's worth to verify with master, because some cases could be originally unsupported as well.

Reply:
Failures on ARM are also fixed. They are concerning cases of empty input. As I know acl of Reduce node doesn't support empty input. Now that we don't use acl kernel in these empty input cases, <makeExecutor> should be avoided.